tools/tools.sh

```diff
@@ -5,19 +5,34 @@
 ##############################################################################
 
 check_utilities () {
-  # Check that jq is installed
-  if ! which jq >/dev/null; then
-    echo "[$0] jq does not exist. Install it from here: https://stedolan.github.io/jq/download/"
-    echo "[$0] Please install jq version 1.5 or above."
-    echo "[$0] Also, please make sure it is in the PATH."
-    exit 1
-  fi
+  echo "[$0] ***** Checking required tools... *****"
+  REQUIRED_TOOLS=("docker" "docker compose" "yq" "jq")
```

Comment:
Some installations use "docker-compose" v2 binary, so the naming may be wrong here if someone's using "docker-compose" instead of "docker compose".
Maybe we should enforce versions checks only for yq and jq for the time being.

Reply:
In fact we could load the env var $DOCKER_COMPOSE_BINARY (which is defined in .env) to determine it.